docker-images/kafka/scripts/kafka_tls_prepare_certificates.sh

```diff
@@ -20,6 +20,16 @@ function create_keystore {
    RANDFILE=/tmp/.rnd openssl pkcs12 -export -in $3 -inkey $4 -chain -CAfile $5 -name $6 -password pass:$2 -out $1
 }
 
+# Parameters:
+# $1: Path to the new keystore
+# $2: Truststore password
+# $3: Public key to be imported
+# $4: Private key to be imported
+# $5: Alias of the certificate
+function create_keystore_for_ext_listener {
+   RANDFILE=/tmp/.rnd openssl pkcs12 -export -in $3 -inkey $4 -name $5 -password pass:$2 -out $1
```

Comment:
We need to be able to (optionally) accomodate the CA file and have support for it. This is because of how trusting the root passes the trust to the leafs of the CA tree. We need to make sure that is handled. Have you tested it? (see #/1549 as an example issue for that).

Reply:
Did this get resolved?

Reply:
I think it should load the chain from the crt file if it is there. Stanislav did some testing, so I would expect it to work.